contracts/governance/extensions/GovernorVotesComp.sol

```diff
@@ -18,14 +18,25 @@ abstract contract GovernorVotesComp is Governor {
         token = token_;
     }
 
+    /**
+     * See EIP 5805.
+     */
+    function clock() public view virtual override returns (uint256) {
```

Comment:
Do you think this function needs to be public? What we really want to add and make public is `CLOCK_MODE` I think, but `clock` serves no use externally I think.

Reply:
I think that the clock used by the governor should be exposed, just like it is exposed by the token. Other contracts might want to connect to a governor, and if we don't expose the `clock()` we are seriously limiting composability.

`CLOCK_MODE` should be a complement on top of `clock()` and not the other way around.